src/css/components/panel.scss

```diff
@@ -40,6 +40,18 @@ $panel-padding: $grid-3;
   flex-direction: row;
   justify-content: flex-start;
   margin: 0;
+
+  p, span {
+    font-size: 0.9rem;
+  }
+}
+
+* + .panel-actions {
+  margin-top: $grid-1;
+}
+
+.panel-actions + * {
+  margin-top: $grid-2;
```

Comment:
I'm not a fan of `*` rules, but given that this is simply adding margin between elements it seems sane enough. I've certainly seen similar elsewhere.